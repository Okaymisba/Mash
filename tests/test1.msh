b = 123;
print(b)

c = 123.456;
print(c)

d = true;
print(d)

e = false;
print(e)

f = 3.142f;
print(f)