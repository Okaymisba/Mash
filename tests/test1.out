123
123.456000
true
false
3.142000